x-pack/plugins/ml/server/models/data_recognizer/modules/network/ml/rare_destination_country.json

```diff
@@ -0,0 +1,35 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "A machine learning job detected an unusual destination country name in the network logs. This can be due to initial access, persistence, command-and-control, or exfiltration activity. For example, when a user clicks on a link in a phishing email or opens a malicious document, a request may be sent to download and run a payload from a server in a country which does not normally appear in network traffic or business work-flows. Malware instances and persistence mechanisms may communicate with command-and-control (C2) infrastructure in their country of origin, which may be an unusual destination country for the source network.",
```

Comment:
As above, for consistency of style with the other security jobs, I'd suggest changing the first part of the description here to 

```
Security: Network - Detects unusual destination country names in the network logs.
```